terraform-plans/templates/github/snap_check.yaml.tftpl

```diff
@@ -0,0 +1,40 @@
+# This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation
+# To update the file:
+# - Edit it in the canonical/solutions-engineering-automation repository.
+# - Open a PR with the changes.
+# - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
+name: Tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+
+concurrency:
+  group: $${{ github.workflow }}-$${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
```

Comment:
maybe we can split the jobs into linting the yaml and another one to properly build the snap